aws-py-stepfunctions/requirements.txt

```diff
@@ -1,2 +1,2 @@
-pulumi>=0.16.4
-pulumi-aws>=0.16.2
+pulumi>=1.0.0
+pulumi-aws>=1.0.0
```

Comment:
Total nit - but should we take this opportunity to be more consistent on using `pulumi-aws` instead of `pulumi_aws`?

(I think that was the direction we planned on standardizing as part of https://github.com/pulumi/templates/pull/48 which never got merged).

Reply:
FWIW, I had been using `pulumi-aws>=1.0.0,<2.0.0` here, since that mirrors what the `^` constraint means in node-land (ignoring 2.0 and later).